morph.go

```diff
@@ -143,3 +176,95 @@ func computePendingMigrations(appliedMigrations []*models.Migration, sourceMigra
 
 	return pendingMigrations, nil
 }
+
+func breakDownMigrations(migrations []*models.Migration, direction models.Direction) ([]*models.Migration, map[string]*models.Migration, error) {
```

Comment:
Shouldn't both `breakDownMigrations` and `findDownScripts` do more or less the same thing but for different directions? Maybe we can think as well on adding some symmetrical names for them if we cannot make them generic (`findDownScripts` and `findUpScripts`, e.g.) or/and adding some doc comments for clarity

I understand how both methods compute the list of migrations in a different way and one requires two sources instead of one, but I think we can create a clear pair of entrypoints with similar names to obtain sorted lists of up and down migrations and then break their logic on different methods for code clarity